libs/application/templates/data-protection-complaint/.eslintrc

```diff
@@ -0,0 +1,249 @@
+{
```

Comment:
Probably best if you use extends here like most of the other apps/libs?

```
{ "extends": "../../.eslintrc", "rules": {}, "ignorePatterns": ["!**/*"] }
```

Reply:
Sure thing, we might wanna do that for the other templates as well then